xml/virt_logs.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-logs">
+ <title>Gathing system information and logs</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  When a virtualization host encounters a problem it is often necessary to collect
+  a detailed system report, which can be done with the help of the
+  <command>supportconfig</command> tool. See <xref linkend="sec-admsupport-supportconfig-sr"/>
+  for more information about <command>supportconfig</command>.
+ </para>
+ <para>
+  In some cases the information gathered by <command>supportconfig</command> is
+  insufficient and logs generated from a custom logging or debugging configuration
+  might be required to determine the cause of a problem.
+ </para>
+ <sect1 xml:id="sec-logs-libvirt">
+  <title>&libvirt; log controls</title>
```

Comment:
I think we want to extend this to answer the user question:

1) Where do I find the libvirt logs in an "out of the box" configuration?
 
2) How do I change the logging configuration?

3) In which cases is more logging required, and how do I properly configure it?

(I would go simplest use case first: logging everything INFO+ to a file seems immediately actionable and useful)

4) how do I configure more advanced logging? (Filters etc) -> separate Advanced Logging section

Reply:
I improved the logging section and hopefully answered these types of user questions.